examples/e12_global_data/src/main.rs

```diff
@@ -79,17 +79,20 @@ struct Handler;
 #[async_trait]
 impl EventHandler for Handler {
     async fn message(&self, ctx: Context, msg: Message) {
-        if msg.content.to_lowercase().contains("owo") {
-            // Since data is located in Context, this means you are also able to use it within events!
-            let count = {
-                let data_read = ctx.data.read().await;
-                data_read.get::<MessageCount>().expect("Expected MessageCount in TypeMap.").clone()
-            };
-
-            // Atomic operations with ordering do not require mut to be modified.
-            // In this case, we want to increase the message count by 1.
-            // https://doc.rust-lang.org/std/sync/atomic/struct.AtomicUsize.html#method.fetch_add
-            count.fetch_add(1, Ordering::SeqCst);
+        // We are verifying if the bot id is the same as the message author id
+        if msg.author.id != ctx.cache.current_user().id  {
```

Comment:
```suggestion
        if msg.author.id != ctx.cache.current_user_id()  {
```
for consistency with the library, as this method is used more than getting the entire user.